packages/gafl-webapp-service/src/handlers/attribution-handler.js

```diff
@@ -9,5 +10,12 @@ import { initialiseAnalyticsSessionData } from '../processors/analytics.js'
  */
 export default async (request, h) => {
   await initialiseAnalyticsSessionData(request)
+
+  if (request.query[UTM.CAMPAIGN] === 'renewals') {
```

Comment:
I think that we need to put the 'renewals' campaign key into a constant, call it `RENEWALS_CAMPAIGN_ID` and put it in constants.js